code/modules/food_and_drinks/food/foods/baked_goods.dm

```diff
@@ -259,6 +259,259 @@
+	filling_color = "#a128c5"
+	tastes = list("cake" = 5, "sweetness" = 1, "plum" = 2)
+	goal_difficulty = FOOD_GOAL_EASY
+
+/obj/item/food/sliceable/pound_cake
+	name = "pound cake"
+	desc = "A condensed cake made for filling people up quickly."
+	icon = 'icons/obj/food/bakedgoods.dmi'
+	icon_state = "pound_cake"
+	slice_path = /obj/item/food/pound_cake_slice
+	slices_num = 5
+	bitesize = 3
+	filling_color = "#c4cab7"
+	list_reagents = list("nutriment" = 60, "vitamin" = 20)
+	tastes = list("cake" = 5, "sweetness" = 5, "batter" = 1)
+	goal_difficulty = FOOD_GOAL_DUPLICATE
+
+/obj/item/food/pound_cake_slice
+	name = "pound cake slice"
+	desc = "A slice of condensed cake made for filling people up quickly."
+	icon = 'icons/obj/food/bakedgoods.dmi'
+	icon_state = "pound_cake_slice"
+	trash = /obj/item/trash/plate
+	filling_color = "#ffffff"
+	tastes = list("cake" = 5, "sweetness" = 5, "batter" = 1)
+	goal_difficulty = FOOD_GOAL_EASY
+
+/obj/item/food/sliceable/pumpkin_spice_cake
+	name = "pumpkin spice cake"
+	desc = "A hollow cake with real pumpkin."
+	icon = 'icons/obj/food/bakedgoods.dmi'
+	icon_state = "pumpkin_spice_cake"
+	slice_path = /obj/item/food/pumpkin_spice_cake_slice
+	slices_num = 5
+	bitesize = 3
+	filling_color = "#ee710a"
+	list_reagents = list("nutriment" = 20, "vitamin" = 10)
+	tastes = list("cake" = 5, "sweetness" = 1, "pumpkin" = 1)
+	goal_difficulty = FOOD_GOAL_DUPLICATE
+
+/obj/item/food/pumpkin_spice_cake_slice
+	name = "pumpkin spice cake slice"
+	desc = "A spicy slice of pumpkin goodness."
+	icon = 'icons/obj/food/bakedgoods.dmi'
+	icon_state = "pumpkin_spice_cake_slice"
+	trash = /obj/item/trash/plate
+	filling_color = "#ee710a"
+	tastes = list("cake" = 5, "sweetness" = 1, "pumpkin" = 1)
+	goal_difficulty = FOOD_GOAL_EASY
+
+/obj/item/food/sliceable/slime_cake
+	name = "Slime cake"
+	desc = "A cake made of slimes. Probably not electrified."
+	icon = 'icons/obj/food/bakedgoods.dmi'
+	icon_state = "slime_cake"
+	slice_path = /obj/item/food/slime_cake_slice
+	slices_num = 5
+	bitesize = 3
+	filling_color = "#0adfee"
+	list_reagents = list("nutriment" = 20, "vitamin" = 10)
+	tastes = list("cake" = 5, "sweetness" = 1, "slime" = 1)
+	goal_difficulty = FOOD_GOAL_DUPLICATE
+
+/obj/item/food/slime_cake_slice
+	name = "slime cake slice"
+	desc = "A slice of slime cake."
+	icon = 'icons/obj/food/bakedgoods.dmi'
+	icon_state = "slime_cake_slice"
+	trash = /obj/item/trash/plate
+	filling_color = "#0adfee"
+	tastes = list("cake" = 5, "sweetness" = 1, "slime" = 1)
+	goal_difficulty = FOOD_GOAL_EASY
+
+/obj/item/food/sliceable/spaceman_cake
+	name = "spaceman's cake"
+	desc = "A spaceman's trumpet frosted cake."
+	icon = 'icons/obj/food/bakedgoods.dmi'
+	icon_state = "trumpet_cake"
+	slice_path = /obj/item/food/spaceman_cake_slice
+	slices_num = 5
+	bitesize = 3
+	filling_color = "#610977"
+	list_reagents = list("nutriment" = 20, "vitamin" = 10, "cream" = 5, "berryjuice", 5)
+	tastes = list("cake" = 4, "violets" = 2, "jam" = 2)
```

Comment:
Tastes like violets?? Am I missing an obvious thing here or... why does it taste like violets? Is it because of the trumpets ingredient which is a subtype of lilies?

Reply:
Yes. It's directly from TG and purposeful to have a floral taste due to the trumpets.